WordPress/twentytwentytwo
# Add code linting and github action

**Description**

This PR is a simpler alternative to #26. (Does not include the pre-commit hook)

**Testing Instructions** 

1. Run `npm install && composer install`
2. Try all of the test commands:
- `npm run lint:css`
- `composer run analyze functions.php`
- `composer run lint`
- `composer run standards:check`
- `composer run standards:fix`
